examples/stocks/lib/i18n/regenerate.md

```diff
@@ -15,7 +15,7 @@ for more info.
 `messages_all.dart`, and `stock_strings.dart` with the following command:
 
 ```dart
-dart ${FLUTTER_PATH}/dev/tools/localization/gen_l10n.dart --arb-dir=lib/i18n \
+dart ${FLUTTER_PATH}/dev/tools/localization/bin/gen_l10n.dart --arb-dir=lib/i18n \
```

Comment:
Is there an implicit TODO here, to move dev/tools/localization apps into the bin directory?

Reply:
I was thinking about this as well -- We should reorganize the `dev/tools/localization` apps.

I'll create corresponding issues to track this and link it to this issue, since I'm not sure if a markdown file is a good place to add a TODO.

Reply:
https://github.com/flutter/flutter/issues/45907